js/metroInit.js

```diff
@@ -33,62 +33,77 @@ UIFreewallVertical = (function() {
 
   UIFreewallVertical.setupFreewalVertical = function(data) {
     var wall = new freewall('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])');
-    var cellWidth;
-    var cellHeight;
+
+    function resizePanels() {
+      var containerWidth = $('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])').parent().width();
+      var $panels = $('.panels.size11, .panels.size21, .panels.size22');
+      var currentClasses = $panels.attr('class');
+      var indexOfContainerClass = currentClasses.indexOf('panel-container');
+
+      if (indexOfContainerClass > -1) {
+        var styleToRemove = currentClasses.substring(indexOfContainerClass);
+        $panels.removeClass(styleToRemove);
+      }
+
+      if (containerWidth >= 1300) {
+        $panels.addClass('panel-container-1300');
```

Comment:
Instead of adding classes to all the panels, why not just add the class to the container, (which I've defined in my suggestion as `$container`)?

You'll just need to change the CSS selectors in the stylesheet to the following (with a space in the middle)/

```css
.panel-container-1200 .panels.size22 {
  /* ... */
}
```

Reply:
Good idea! Fixed.